yt_dlp/extractor/trovo.py

```diff
@@ -229,17 +259,29 @@ class TrovoChannelVodIE(TrovoChannelBaseIE):
         'url': 'trovovod:OneTappedYou',
         'playlist_mincount': 24,
         'info_dict': {
-            'id': '100719456',
+            'id': 100719456,
```

Comment:
id should be string

Reply:
Fixed, wasn't sure about the expected type in info dict but the test was complaining about a mismatch.